Makefile

```diff
@@ -49,8 +48,15 @@ unit-test-wasm: depend
 	@scripts/check_unit_wasm.sh
 
 .PHONY: bdd-test
-bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker
-	@scripts/check_integration.sh
+bdd-test: clean generate-test-keys agent-rest-docker sample-webhook-docker bdd-js bdd-go
+
+.PHONY: bdd-go
+bdd-go:
```

Comment:
would make sense to match prefix with the parent target. e.g., bdd-test-go & bdd-test-js

Reply:
done